x-pack/plugins/observability/public/pages/alerts/containers/alerts_page/alerts_page.tsx

```diff
@@ -263,9 +268,9 @@ function AlertsPage() {
             data-test-subj="statDisabled"
           />,
           <EuiStat
-            title={ruleStats.muted}
+            title={ruleStats.muted + ruleStats.snoozed}
             description={i18n.translate('xpack.observability.alerts.ruleStats.muted', {
```

Comment:
@XavierM Shall we make it explicit in the name that it contains muted and snoozed `xpack.observability.alerts.ruleStats.mutedAndSnoozed`?

Reply:
I don't mind keeping it as it is either.

Reply:
let's keep it like that for now because this is going to change again in the next release meaning muted will become deprecated sooner than later.